src/components/Footer/Footer.js

```diff
@@ -47,11 +47,13 @@ const Footer = ({ noSocial, noLinks, campaign, copy }) => {
 
 Footer.propTypes = {
   campaign: propTypes.string.isRequired,
+  fallbackMenu: propTypes.array,
```

Comment:
Given that we need this array prop to be a certain structure, I think we're best off defining it properly, something like:

```
   fallbackMenu: PropTypes.arrayOf(PropTypes.shape({
     url: PropTypes.string.isRequired,
     title: PropTypes.string.isRequired,
   })).isRequired,
```

Reply:
Done